docs/content/TestData.fsx

```diff
@@ -208,24 +208,58 @@ all singular elements returned is the same tuple.*)
 need to use `Gen.constant` often. Still, it can come in handy if you need to
 keep the value of a particular type constant while you vary other values.
 
+#### Choose
+
+You can use the `Gen.choose` function to create a generator of singular integer
+values between a minimum and maximum value, both inclusive:*)
+
+(***define-output:ChooseBetweenZeroAndNineExample***)
+Gen.choose (0, 9) |> Gen.sample 0 10
+
+(**This example generates a single integer value between 0 and 9. Since
+`Gen.constant` doesn't rely on the `size` argument, it's `0` in this example,
```

Comment:
Well spotted :smile: Pull request updated.

> Do we need to repeat this same line every time?

We don't need to, but my approach so far has been to write each section to be as self-contained as possible. The intent is that the documentation can act as a reference more than a narrative.

It does lead to repetition, but I think it's safer than assuming that the reader has read everything from the top.

I'll try to vary my prose a bit in the future :flushed: